main.go

```diff
@@ -205,7 +206,8 @@ func main() {
 			message = string(tmp)
 		}
 		oktetoLog.Fail(message) // TODO: Change to use ioController  when we fully move to ioController
-		if uErr, ok := err.(oktetoErrors.UserError); ok {
+		var uErr oktetoErrors.UserError
+		if errors.As(err, &uErr) {
```

Comment:
If a userError was being wrapped, we were not showing the hint. As the error message is being displayed in line `208`, if the error chain contains an `UserError` we should display the Hint.

Raising awareness because it might happen that some hints start to appear after these changes, but I think it is not a problem.